packages/core/src/initializer/stryker-initializer.ts

```diff
@@ -1,17 +1,17 @@
-import * as child from 'child_process';
+import child from 'child_process';
```

Comment:
can't we keep it consistent? childProcess for example

Reply:
yes definitely. Good point.